### ISSUE TITLE
Item shows up as suspicious stew in inventory, but correctly in the hand and when dropped

### ISSUE THREAD

**Issue by TerdyTheTerd** [Daniel-Tomov/AddHearts#3]
Unsure if its a missing texture, but the display inside an inventory does not have a heart texture, its just the underlying stew item

**Comment by TerdyTheTerd** [comment]
This seems to be resolved with the new 3.0 release.

**Comment by Daniel-Tomov** [comment]
Thank you, closing.

**Comment by TerdyTheTerd** [comment]
Well, this issue was resolved, but after updated to 3.1 today it's no longer loading in the texture in the inventory. I thought it was an issue with Nexo, so I removed that plugin and added the resource pack directly to my client, but the item texture is not loading:
![image](https://github.com/user-attachments/assets/5517000c-87f4-45d2-88a9-03c56672601c)

Any ideas?

**Comment by Daniel-Tomov** [comment]
No, I have not seen this issue with this resource pack before. I tested running the resource pack on both Windows and Linux and there were no issues with the heart texture not displaying in the GUI. When I developed this for 1.18 two years ago, I do not remember having issues with the texture. I am confused as to why Minecraft is not displaying the texture when the code for dispalying the texture is in the resource pack files.
https://github.com/Daniel-Tomov/AddHearts/blob/720623ece62a1efd5f259d74ad4c36950cbba107/ResourcePack/ResourcePack/assets/minecraft/models/item/heart/1.json#L3094
